doc/policies/admin.rst

```diff
@@ -411,6 +411,33 @@ type: bool
 If the policy ``tokengroup_delete`` is defined, the administrator is allowed to
 delete existing tokengroup definitions.
 
+.. _policy_serviceids:
+
+serviceid_add
+~~~~~~~~~~~~~
+
+type: bool
+
+.. index:: service ID
+
+This policy allows the administrator to add a new service ID to the list of
+defined services. These service ID can then be used in attaching SSH keys.
+See :ref:`application_ssh`.
```

Comment:
Ah I see, since the application specific token also uses a service_id. However, this is not implemented, yet.
In the application specific token the service_id is currently a free text string.
I opened a new issue for that #3585 
Will not do in this PR.